modules/ducktests/tests/ignitetest/services/utils/ignite_configuration/data_storage.py

```diff
@@ -56,3 +57,6 @@ class DataStorageConfiguration(NamedTuple):
     wal_page_compression: str = None
     wal_page_compression_level: int = None
     wal_path: str = None
+    write_recovery_data_on_checkpoint: bool = False
```

Comment:
Please use None here which means "use default by not specifiing in config XML file".

This will allow to run existing tests against old versions.
And also the persistence_upgrade_test which uses different versions in a single run.

Reply:
Fixed